app/themes/singular/views/devise/registrations/new.html.erb

```diff
@@ -3,21 +3,19 @@
 <% end %>
 
 <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), validate: true) do |f| %>
-    <%= devise_error_messages! %>
-
-    <%= f.input :email, autofocus: true %>
-    <%= f.input :name %>
-    <div class="select-image text-center">
-      <%= f.attachinary_file_field :avatar if cloudinary_enabled? %>
-      <%= f.file_field :profile_image, label: "Avatar" unless cloudinary_enabled? %>
-    </div>
-    <%= f.input :company %>
-    <%= f.input :title %>
-    <% if @validatable %>
-    <!--<em>(<%= @minimum_password_length %> characters minimum)</em>-->
-    <% end %>
-    <%= f.input :password, autocomplete: "off" %>
-    <%= f.input :password_confirmation, autocomplete: "off" %>
-
-    <%= f.submit I18n.t('devise.registrations.new.sign_up'), class: 'btn btn-warning' %>
+  <%= devise_error_messages! %>
+  <%= f.input :email, autofocus: true %>
+  <%= f.input :name %>
+  <div class="select-image text-center" id='test1'>
+    <%= f.attachinary_file_field :avatar if cloudinary_enabled? %>
+    <%= f.file_field :profile_image, class: 'user-avatar', 'data-content' => I18n.t('devise.registrations.choose_avatar'), label: "Avatar" unless cloudinary_enabled? %>
```

Comment:
This looks great when you view the screen(s) but it no longer shows the name of the file selected next to the button.  In fact there is no way to know that you have picked an avatar at all (except for the green checkmark).